client/src/pages/GameSelect.jsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import GameListItem from '../components/GameListItem';
 
-const GameSelect = () => <div className="App">Game Select</div>;
+const propTypes = {
+  gameList: PropTypes.arrayOf(PropTypes.shape({})),
+  onJoinClick: PropTypes.func.isRequired,
+  onCreateGame: PropTypes.func.isRequired,
+};
 
+const defaultProps = {
+  gameList: [],
+};
+
+const noGames = onCreateGame => (
+  <React.Fragment>
+    <h2>No Games Made, be the first!!</h2>
+    <button type="button" onClick={onCreateGame}>Create Game</button>
+  </React.Fragment>
+);
+
+const GameSelect = ({ gameList, onJoinClick, onCreateGame }) => (
+  <React.Fragment>
+    <h1> Game List</h1>
+    {
+      gameList.length <= 0
+        ? noGames(onCreateGame)
+        : gameList.map(game => <GameListItem key={game.id} {...game} onJoinClick={onJoinClick} />)
```

Comment:
`{...game}` is "spreading" all the keys and values of that object into `<GameListItem `. so if the Game object has those props, it will send them in. The game is an entry in the `gameList` prop array. Typically you only do prop type validations for the pieces we use. Since we need to call map on it, we prop validate that it is an array, but we aren't using props from it so the shape is just `{}` object.

Since I did add `key={game.id}` though, i probably need to prop validate the id is there and is a string